hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/TopicEndpoint.java

```diff
@@ -18,11 +18,13 @@
 import javax.ws.rs.core.Response;
 import java.util.List;
 
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 import static pl.allegro.tech.hermes.api.AvroMediaType.AVRO_BINARY;
 
 @Path("topics")
 public interface TopicEndpoint {
+
+    String APPLICATION_JSON = "application/json;charset=UTF-8";
```

Comment:
any reason for this change?

Reply:
It is compliant with default HermesClient's headers

Reply:
I think this change won't be necessary, I will revert it

Reply:
thanks